### PR TITLE
[Non-Modular] Nekomimetic Change

### PR DESCRIPTION
**Pull request by Nerev4r**
## About The Pull Request
To make the language less of a meme, I've replaced the syllables in there with ones that aren't weebspeak. Instead, cat sounds.
image
As you can see this is moderately painful to read so I'm instadrafting this so I can get a better syllable list together.

## How This Contributes To The Skyrat Roleplay Experience
1) it's a language entirely designed around being funny rather than being immersive
2) it has made 100x less sense ever since we added Yangyu because it means that speakers of actual real Japanese cannot understand speakers of 'about 14 japanese words and all of them are anime shit'.
3) it is straining at the immersion to have people say stuff like 'she has a nekomimetic accent' or 'when he signs in nekomimetic' considering it is, again, abt 14 japanese words and they're all weeb buzzwords.
4) this has wider applicability for feline species at large.

## Changelog
:cl:
add: Nekomimetic has been replaced with actual cat sounds.
/:cl:

### CODE REVIEW

**Comment by Nerev4r** [comment]
hello this is day 1 nerevar here to ask for some cat syllables. pls submit.

File: code/modules/language/nekomimetic.dm [hunk 1]
```diff
@@ -1,14 +1,25 @@
 /datum/language/nekomimetic
 	name = "Nekomimetic"
-	desc = "To the casual observer, this langauge is an incomprehensible mess of broken Japanese. To the felinids, it's somehow comprehensible."
+	//ORIGINAL: desc = "To the casual observer, this langauge is an incomprehensible mess of broken Japanese. To the felinids, it's somehow comprehensible."
+	desc = "Cat vocalizations, categorized by their acoustic qualities, are quite complicated; both linguistically and emotionally. While this mess of an array of sounds may come off as utter nonsense to outsiders, to felines, it's quite comprehensible."
```
Comment (tf-4) [inline code/modules/language/nekomimetic.dm:4]:
```suggestion
	desc = "Cat vocalizations, categorized by their acoustic qualities, are quite complicated both linguistically and emotionally. While this mess of an array of sounds may come off as utter nonsense to outsiders, to felines it's quite comprehensible."
```

**Comment by YakumoChen** [comment]
This was PRed as grudgecode towards me when I rejected the idea when pinged about it on discord.

Furthermore, humans with cat features speaking a mutated human language makes much more sense than them developing a bunch of cat noises as a language (remember, they're still mostly human).

This PR feels like a slight against a large percentage of players who enjoy playing felinids in general (which would be the weeb crowd which the language is, in part, a homage to)

Nekomimetic makes sense from just a purely logical standpoint of how such a race would develop. It's much more sensible that a bunch of otaku / weebs would be the first to jump on the genemodding bandwagon and, if we're going by the TG lore (the only lore we have on the race), becoming a race outcasted from their peers and mutating their favoured language (Japanese) into something different and incompatible is both completely plausible and happens in reality all the time (see: the development of almost every other language on the planet)

**Comment by YakumoChen** [comment]
> it is straining at the immersion to have people say stuff like 'she has a nekomimetic accent' or 'when he signs in nekomimetic' considering it is, again, abt 14 japanese words and they're all weeb buzzwords.

This reasoning is insufficient, as the "14 words" is purely a limitation of the coding for the language and does has no actual bearing on how the language would actually sound like otherwise.

> it has made 100x less sense ever since we added Yangyu because it means that speakers of actual real Japanese cannot understand speakers of 'about 14 japanese words and all of them are anime shit'.

Languages develop exactly like this all the time. Remember: English is a germanic language, but that doesn't mean I can suddenly understand everything in German when I hear it.

> this has wider applicability for feline species at large.

Code a language for those species instead of maliciously removing the existing one for this species.

**Pull request closed without merging**